tests/unit_tests/backends/test_freezing_methods.py

```diff
@@ -96,8 +96,12 @@ def test_freeze_singular(backend_class):
         )
 
     @staticmethod
+    @pytest.mark.parametrize("double_precision", (True, False))
     # pylint: disable=too-many-locals
-    def test_freeze_time_dependent(backend_class, plot=False):
+    def test_freeze_time_dependent(backend_class, double_precision, plot=False):
+        if backend_class.__name__ == "Numba" and not double_precision:
+            pytest.skip()
```

Comment:
maybe let's create an issue for that?

Reply:
good point! https://github.com/open-atmos/PySDM/issues/1144 (will be adding TODO labels as well)